tests/modules/images/webp-uploads/load-tests.php

```diff
@@ -801,6 +802,7 @@ public function it_should_create_webp_for_some_sizes_which_are_smaller_in_webp_f
 		$this->opt_in_to_jpeg_and_webp();
 
 		add_filter( 'webp_uploads_discard_larger_generated_images', '__return_true' );
+		remove_all_filters( 'wp_editor_set_quality' );
```

Comment:
See above, why is this necessary?

Reply:
same case as above, webp images generated is smaller in size.

Reply:
See above on how I think we can solve this in a more reliable way that doesn't assume as much about the defaults.